tests/integration/census/test_census_individual_submission_data.py

```diff
@@ -21,122 +21,122 @@ def get_expected_submission_data():
             {'answer_id': 'middle-names',
              'answer_instance': 0,
              'group_instance': 0,
-             'group_instance_id': 'household-member-3',
+             'group_instance_id': None,
```

Comment:
Should these be `None` or should we just omit the key?

Reply:
They are set to None by https://github.com/ONSdigital/eq-survey-runner/blob/c86bb548a507694ff2c7efb26075f57327528e91/app/data_model/answer_store.py#L17, however this could be changed?

Reply:
I spoke with @dwyeradam about this and it will make the answer comparison more complex if the `group_instance_id` key/attr is missing some of the time, so suggest leaving it in. Could filter it out in submission dumping/flushing perhaps?